class/upload.php

```diff
@@ -84,8 +84,9 @@ public function upload_file( $image_only = false ) {
         //$wpuf_file = isset( $_FILES['wpuf_file'] ) ? array_map( 'sanitize_text_field', wp_unslash( $_FILES['wpuf_file'] ) ) : []; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
         $wpuf_file = isset( $_FILES['wpuf_file'] ) ? $_FILES['wpuf_file'] : []; // WPCS: sanitization ok.
 
+        $file_extension = pathinfo( $wpuf_file['name'], PATHINFO_EXTENSION );
         $upload = [
-            'name'     => $wpuf_file['name'],
+            'name'     => wp_hash( time() ) . '.' . $file_extension,
```

Comment:
Filename should be like myfile-3a401f66.png. Generate hash like this:
```
$hash = wp_hash( time() );
$hash = substr( $hash, 0, 8 );
```
Then use it like this,
```
'name' => $wpuf_file['name'] . '-' . $hash . '.' . $file_extension
```